src/kudu/consensus/persistent_vars.cc

```diff
@@ -0,0 +1,150 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+#include "kudu/consensus/persistent_vars.h"
```

Comment:
this should probabaly move down to keep in sync with the style - include standard headers first and then kudu headers

Reply:
This one is consistent w/ the Google style guide since this file is an implementation of persistent_vars.h: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes